HISTORY.md

```diff
@@ -22,13 +21,14 @@ http://visjs.org
 - FIX #3163: Prevent crashes from invalid id's in `Clustering.findNode()`
 - FIX #3106: Ensure start and end of stabilization progress events is sent
 - FIX #3015: Properly handle newline escape sequences in strings for DOT
-- FIX Refactoring of LayoutEngine
+- FIX: Refactoring of LayoutEngine (#3110)
```

Comment:
That doesn't look like the other entries. I'd expect `FIX #3110: Refactoring of LayoutEngine`